### PR TITLE
Fix/frontend40 sidebar studio

### PR DESCRIPTION
**Pull request by so5tmaker**

### CODE REVIEW

**Comment by so5tmaker** [comment]
[Implement sidebar component in studio app](https://trello.com/c/zg6s8Grx/40-implement-sidebar-component-in-studio-app)

File: frontend/src/components/studio/sidebar/sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  return (
+    <aside className={styles.sidebar}>
+      <Link to={AppRoute.ROOT}>
+        <div className={styles.button}>
+          <Icon name={IconName.LOGOTIP} />
+        </div>
+      </Link>
+
+      {sideBarItems.map((item: ISideBarItem) => (
+        <Link to={item.routeName as AppRoute}>
```
Comment (kyrylo-burmelov) [inline frontend/src/components/studio/sidebar/sidebar.tsx]: https://en.reactjs.org/docs/lists-and-keys.html

File: frontend/src/components/studio/sidebar/sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import { IconName, IconColor, AppRoute } from 'common/enums/enums';
+import cn from 'clsx';
+import { ISideBarItem, sideBarItems } from './config';
+import { useLocation } from 'hooks/hooks';
+
+import styles from './styles.module.scss';
+
+const StudioSidebar: FC = () => {
+  const { pathname } = useLocation();
+  const getColor = (item: string): string => (pathname === item ? IconColor.GREEN : IconColor.WHITE);
```
Comment (kyrylo-burmelov) [inline frontend/src/components/studio/sidebar/sidebar.tsx]: Not understandable enough. Better to do it in thie way:

```
const isActiveTab = ...

// then add the color using isActiveTab var 
```

Comment (kyrylo-burmelov) [reply frontend/src/components/studio/sidebar/sidebar.tsx]: In addition, your solution will not work when you have bigger route name. Like:

`/home/route-name/route-name`

File: .stylelintrc.yml [hunk 3]
```diff
@@ -13,6 +13,8 @@ rules:
   unit-disallowed-list:
     - em
     - rem
+  at-rule-no-unknown: null
+  scss/at-rule-no-unknown: true
```
Comment (kyrylo-burmelov) [inline .stylelintrc.yml]: As we discussed in slack. Try to get rid of that and use extends scss. Let me know if it does not work

**Review by kyrylo-burmelov — CHANGES_REQUESTED** [review]
Review is ready) Let me know if you have any questions

**Review by kyrylo-burmelov — APPROVED** [review]

**Pull request merged**